### DISCUSSION TITLE
CSS 選擇器小遊戲

### DISCUSSION THREAD

**Discussion by stan890425** [1101-GS4719-C/Python-Community-GS4719-C#613] [General]
老師上禮拜有跟我們介紹一個CSS 選擇器的小遊戲，
我上禮拜上課就玩上癮了~~
一路把所有的關卡破完了，
中間有一點小卡關，所以去找了攻略，總算把它破玩了~~

下面就是我找到攻略:
https://medium.com/@tonylin1208/css-%E9%81%B8%E6%93%87%E5%99%A8%E5%B0%8F%E9%81%8A%E6%88%B2-d3c36fe471f2

**Comment by 11timyyy** [comment]
動畫做的很可愛，也非常容易上手，邊完還可以邊學!

**Comment by awkward-willy** [comment]
我自己也花了很多時間在玩這個！
整個動畫非常的魔性，遊戲的過程也可以學到不少
大推

**Comment by HsuChen88** [comment]
我也一直卡關好難過

**Comment by jason79461385** [comment]
謝謝分享，之前自己在玩的時候遇到卡關的就放棄了

**Comment by RogelioKG** [comment]
真心認為這樣子互動式遊戲的學習方式上手很快，相對於閱讀大量文字，效率更高。
遊戲確實是很棒的交互媒介。這種互動學習方式才是未來發展的方向。

**Comment by Chiuhsianglin** [comment]
系上修的網頁程式設計也有用到css，但在玩老師的遊戲時會發現自己好像對他有點陌生了，會忘記有些code怎麼打，所以我認為老師的遊戲可以很好的幫助我對css更熟，更記得它。

**Comment by poweichen00** [comment]
我覺得用遊戲學比較不會那麼的痛苦，也非常簡易，可以學到不少css。

**Comment by Evening218** [comment]
我很喜歡這個選擇器小遊戲!動畫給人一種很療癒的感覺,而且我覺得它讓我比較理解css了,雖然我也是玩一玩就卡關,只好上網找攻略,但是真的很有趣
